assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -73,6 +79,18 @@ const MiniCartBlock = ( {
 		}
 	}, [ isOpen, cartIsLoading, cartItems.length, emptyCartRef ] );
 
+	const ariaLabel = sprintf(
+		/* translators: %1$d is the number of products in the cart. %2$s is the cart total */
+		_n(
+			'%1$d item in cart, total price of %2$s',
+			'%1$d items in cart, total price of %2$s',
+			cartItemsCount,
+			'woo-gutenberg-products-block'
+		),
+		cartItemsCount,
+		formatPrice( cartTotals.total_items )
```

Comment:
If you set your settings as follows:

1. In WooCommerce > Settings, check this option: 'Enable tax rates and calculations'.
2. In the Taxes tab, select 'No, I will enter prices exclusive of tax' and set 'Display prices during cart and checkout' to 'Including tax'.

You will notice taxes are not included in the accessibility text (but they are in the Storefront legacy mini cart):

![prices-mini-cart](https://user-images.githubusercontent.com/3616980/135054034-2651af9b-03aa-4ea7-9a3a-e352754213f9.png)
_(Storefront mini cart on the left / code of the Mini Cart block on the right)_

I think we should mimic the Storefront mini cart here, so I would use `cartTotals.total_price` instead of `total_items`.

Besides that, why not passing the currency data here? It's included in the `cartTotals` object, so I think it's better if we pass it. I guess you could do something like this:

```suggestion
		formatPrice( cartTotals.total_price, getCurrencyFromPriceResponse( cartTotals ) )
```

Reply:
I didn't notice we have `getCurrencyFromPriceResponse`, thanks for the head-up!